scripts/zones/Windurst_Waters/npcs/Leepe-Hoppe.lua

```diff
@@ -201,6 +225,17 @@ function onEventFinish(player,csid,option)
         player:setCharVar("MissionStatus",1);
     elseif (csid == 742) then
         finishMissionTimeline(player,3,csid,option);
+
+    -- Tuning In
+    elseif csid == 884 then
+        player:addQuest(WINDURST,tpz.quest.id.windurst.TUNING_IN)
+
+    elseif csid == 886 then
+        player:tradeComplete()
+        npcUtil.completeQuest(player, WINDURST, tpz.quest.id.windurst.TUNING_IN, {
+            gil = 4000,
```

Comment:
gil = 4000 * GIL_RATE,

For custom servers with custom gil rate

Reply:
I believe the `completeQuest` function handles this already:

https://github.com/project-topaz/topaz/blob/0e8fdf88df9f644fb47d037ab253e97f69cdbb84/scripts/globals/npc_util.lua#L361-L364